egs2/voxceleb/spk1/conf/train_RawNet3.yaml

```diff
@@ -0,0 +1,35 @@
+# RawNet3 reproduce recipe configuration.
+encoder: rawnet3
+#encoder_conf:  # uncomment for customized arguments
+
+#model_conf:    # uncomment for customized arguments
+
+frontend: raw
+
+pooling: chn_attn_stat
+pooling_conf:
+  input_size: 1536
+  output_size: 3072
+
+projector: rawnet3
+projector_conf:
+  input_size: 3072
```

Comment:
It might be better to add a comment to describe how this value is calculated according to the dimension in other modules.

Reply:
Similarly, for other arguments that have an implicit relationship (such as `X = Y * 1.5`), it's better to mention it in the corresponding configuration lines.